RecoJets/JetProducers/interface/PileupJetIdAlgo.h

```diff
@@ -64,16 +64,22 @@ class PileupJetIdAlgo {
 	PileupJetIdentifier internalId_;
 	variables_list_t variables_;
 
-	std::unique_ptr<TMVA::Reader> reader_, reader_jteta_0_2p5_, reader_jteta_2p5_2p75_, reader_jteta_2p75_3_, reader_jteta_3_5_;
-	std::string    tmvaWeights_, tmvaWeights_jteta_0_2p5_, tmvaWeights_jteta_2p5_2p75_, tmvaWeights_jteta_2p75_3_, tmvaWeights_jteta_3_5_, tmvaMethod_; 
-	std::vector<std::string>  tmvaVariables_, tmvaVariables_jteta_0_3_, tmvaVariables_jteta_3_5_;
+	std::unique_ptr<TMVA::Reader> reader_;
+        std::vector<std::unique_ptr<TMVA::Reader>> etaReader_;
+	std::string    tmvaWeights_, tmvaMethod_;
+        std::vector<std::string> tmvaEtaWeights_;
+	std::vector<std::string>  tmvaVariables_;
+        std::vector<std::vector<std::string>> tmvaEtaVariables_;
 	std::vector<std::string>  tmvaSpectators_;
 	std::map<std::string,std::string>  tmvaNames_;
 	
 	Int_t   version_;
 	Float_t impactParTkThreshod_;
 	bool    cutBased_; 
 	bool    etaBinnedWeights_;
+        Int_t   nEtaBins_;
+        std::vector<Double_t> jEtaMin_;
+        std::vector<Double_t> jEtaMax_;
```

Comment:
can we change everything in this file to C/C++ native types?
int, float, double